javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -87,7 +87,7 @@ public JavadocInlineTag(Type type, String content) {
 
     @Override
     public String toText() {
-        throw new UnsupportedOperationException();
+        return "{@" + type.keyword + this.content;
```

Comment:
Yeah it would be nice. The content has the closing brace. I will check if I can extract it.